.buildkite/scripts/common.sh

```diff
@@ -4,25 +4,24 @@
 
 # Temporary artifacts download directory.
 ARTIFACTS_CACHE_DIR=/tmp/artifacts
-ARTIFACTS_TMP_DIR=/tmp/artifacts_tmp
 CLEANING_UP=0
 
 # Download an artifact and change its mode
 download_artifact() {
-    local name=$1
-    local dst_dir=$2
+    local name=${1}
+    local dst_dir=${2}x
```

Comment:
x?

Reply:
Debugging https://github.com/buildkite/agent/issues/2150. There is some magic happening, because using a different name works...